### ISSUE TITLE
Table component is missing focus in Safari

### ISSUE THREAD

**Issue by Maren-SSB** [PxTools/pxtools-project#187]
Tables with horizontal scrolling cannot be accessed or scrolled using the keyboard in Safar, because the table does not receive focus.

## Acceptance criteria

- Horizontally scrollable table containers can receive keyboard focus in all browsers
- Users can scroll horizontally using the keyboard when the container is focused
